src/dtgtk/thumbtable.c

```diff
@@ -2601,6 +2601,7 @@ static gboolean _filemanager_key_move(dt_thumbtable_t *table, dt_thumbtable_move
   int baseid = dt_control_get_mouse_over_id();
   // let's be sure that the current image is selected
   if(baseid > 0 && select) dt_selection_select(darktable.selection, baseid);
+  else if(baseid <= 0) baseid = dt_selection_get_first(darktable.selection);
```

Comment:
We may have unwanted effect in the case where the first selected image is not shown on screen. We may want to add something in order to ensure that we don't go outside the current table...
Note that this may interfere with @Solarer PR #10771

tbh I wasn't even aware of the alt-arrow feature...

Reply:
reverted.